mysql-test/suite/galera/r/galera_ftwrl_drain.result

```diff
@@ -2,6 +2,8 @@ CREATE TABLE t1 (id INT PRIMARY KEY) ENGINE=InnoDB;
 SET GLOBAL wsrep_provider_options = 'dbug=d,apply_monitor_slave_enter_sync';
 INSERT INTO t1 VALUES (1);
 SET SESSION wsrep_sync_wait = 0;
+SET SESSION wsrep_on = 0;
```

Comment:
So, previous value of wsrep_on should be recorded first, then assigned back afterwards.